1572609390/offsets.json

```diff
@@ -0,0 +1,220 @@
+{
+  "portrait": {
+    "Google Pixel 3": {
+      "offset": "+55+199",
+      "width": 1080
+    },
+    "Google Pixel 3 XL": {
+      "offset": "+69+71",
+      "width": 1440
+    },
+    "Google Pixel 4": {
+      "offset": "+44+145",
+      "width": 1080
+    },
+    "Google Pixel 4 XL": {
+      "offset": "+60+193",
+      "width": 1440
+    },
+    "HTC One A9": {
+      "offset": "+78+292",
+      "width": 1080
+    },
+    "HTC One M8": {
+      "offset": "+72+267",
+      "width": 1080
+    },
+    "Huawei P8": {
+      "offset": "+60+230",
+      "width": 1080
+    },
+    "MacBook Air 13\"": {
+      "offset": "+373+123",
+      "width": 2560
+    },
+    "Macbook": {
+      "offset": "+380+128",
+      "width": 2304
+    },
+    "Macbook Pro 13\"": {
+      "offset": "+394+143",
+      "width": 2560
+    },
+    "Macbook Pro 15\"": {
+      "offset": "+441+143",
+      "width": 2880
+    },
+    "Motorola Moto E": {
+      "offset": "+55+185",
+      "width": 540
+    },
+    "Motorola Moto G": {
+      "offset": "+55+189",
+      "width": 720
+    },
+    "Nexus 4": {
+      "offset": "+41+185",
+      "width": 768
+    },
+    "Nexus 5x": {
```

Comment:
I tried to build both PRs locally and use frameit on android , and got this error:
```
[⠦] Framing screenshot './output/my_screenshot.png' [21:18:16]: Tried looking for offset information for 'portrait', Nexus 5X in '/Users/ricardobelchior/.fastlane/frameit/latest/offsets.json'
[21:18:16]: Could not find offset_information for './output/my_screenshot.png'
```
I fixed the issue , changing this offset to: `Nexus 5X` (capital X)